src/devices/ikea.ts

```diff
@@ -658,7 +658,14 @@ const definitions: Definition[] = [
     // #endregion blinds
     // #region appliances
     {
-        zigbeeModel: ['STARKVIND Air purifier', 'STARKVIND Air purifier table'],
+        zigbeeModel: ['STARKVIND Air purifier table'],
+        model: 'E2006',
+        vendor: 'IKEA',
+        description: 'STARKVIND air purifier table',
+        extend: [addCustomClusterManuSpecificIkeaUnknown(), addCustomClusterManuSpecificIkeaAirPurifier(), ikeaAirPurifier(), identify(), ikeaOta()],
+    },
+    {
+        zigbeeModel: ['STARKVIND Air purifier'],
```

Comment:
It may be better to use a `whiteLabel` here [example](https://github.com/Koenkk/zigbee-herdsman-converters/blob/7df9894acb4b904cc1f5a3799d9798ec86eb0edd/src/devices/custom_devices_diy.ts#L277)

Reply:
Thank you for the pointer!

Reply:
Please also add the fingerprint.modelId part for proper detection.

Reply:
Done. Apologies, I missed that somehow.